src/libstd/array.rs

```diff
@@ -8,19 +8,48 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-//! The fixed-size array type (`[T; n]`).
+//! A fixed-size array is denoted `[T; N]` for the element type `T` and
+//! the compile time constant size `N`. The size should be zero or positive.
```

Comment:
Apparently [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt). I just had to google it to understand it's their text for defining what MUST, SHOULD etc mean in their standards